docs/reference/installation/_interfaces/vector-installer.cue

```diff
@@ -0,0 +1,57 @@
+package metadata
+
+installation: _interfaces: "vector-installer": {
+	title:       "Vector Installer"
+	description: """
+		The [Vector installer](\(urls.vector_installer)) is a simple shell
+		script that facilitates that installation of Vector on a variety of
+		systems. It is an unobtrusive and simple option since it installs the
+		`vector` binary in your current direction.
```

Comment:
directory

Doesn't it actually install Vector into `$HOME/.vector/bin` and add that to your path?